pkg/cable/strongswan/getconnections.go

```diff
@@ -7,6 +7,9 @@ import (
 	v1 "github.com/submariner-io/submariner/pkg/apis/submariner.io/v1"
 )
 
+const ConnectingState = "CONNECTING"
+const EstablishedState = "ESTABLISHED"
+
```

Comment:
The strongSwan code has been removed ;-).

Reply:
Oops. Done